sys/include/net/ng_rpl/dodag.h

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (C) 2013  INRIA.
+ * Copyright (C) 2015 Cenk Gündoğan <cnkgndgn@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup net_ng_rpl
+ * @{
+ *
+ * @file        ng_rpl_dodag.h
```

Comment:
Remove specific file here? It's not correct and excludes the link to the file on the RPL source doc page. (I'm trying to learn/experiment with the code. It's good when people come out of the woodwork to help, right? :-))